src/main.ts

```diff
@@ -59,17 +65,10 @@ Phaser.GameObjects.NineSlice.prototype.setPositionRelative = setPositionRelative
 Phaser.GameObjects.Text.prototype.setPositionRelative = setPositionRelative;
 Phaser.GameObjects.Rectangle.prototype.setPositionRelative = setPositionRelative;
 
-document.fonts.load("16px emerald").then(() => document.fonts.load("10px pkmnems"));
-// biome-ignore lint: TODO
-let game;
 // biome-ignore lint: TODO
-let manifest;
-
-const startGame = async () => {
+async function startGame(manifest?: Record<string, string>): Promise<void> {
```

Comment:
```suggestion
async function startGame(gameManifest?: Record<string, string>): Promise<void> {
```
To resolve the `noShadow` lint.